src/schema/v2/artist/helpers.ts

```diff
@@ -163,6 +164,11 @@ export const ARTIST_INSIGHT_MAPPING: Record<
         count === 1 ? "a top industry award" : `${count} top industry awards`
       }`,
   },
+  FOUNDATIONS: {
+    getDescription: () => null,
```

Comment:
would be nice to have a description to understand what's this is about

Reply:
There are already some existing fields without descriptions. I don't think this is blocking, and it can be addressed incrementally. [More chatter](https://artsy.slack.com/archives/C05EEBNEF71/p1724159834359899).